man/rgbasm.5

```diff
@@ -414,8 +414,8 @@ Most of them return a string, however some of these functions actually return an
 .It Fn STRIN str1 str2 Ta Returns the first position of Ar str2 No in Ar str1 No or zero if it's not present Pq first character is position 1 .
 .It Fn STRRIN str1 str2 Ta Returns the last position of Ar str2 No in Ar str1 No or zero if it's not present Pq first character is position 1 .
 .It Fn STRSUB str pos len Ta Returns a substring from Ar str No starting at Ar pos No (first character is position 1, last is position -1) and Ar len No characters long. If Ar len No is not specified the substring continues to the end of Ar str .
-.It Fn STRUPR str Ta Returns Ar str No with all letters in uppercase.
-.It Fn STRLWR str Ta Returns Ar str No with all letters in lowercase.
+.It Fn STRUPR str Ta Returns Ar str No with all ASCII letters in uppercase.
```

Comment:
It may not be clear what ASCII is, so maybe expliciting the range would be better?

```suggestion
.It Fn STRUPR str Ta Returns Ar str No with all ASCII letters (`a-z`) in uppercase.
```

We'd probably want to make that range uppercase for `STRLWR`.